package.json

```diff
@@ -39,20 +39,21 @@
     "whatwg-fetch": "^2.0.3"
   },
   "devDependencies": {
+    "@babel/cli": "^7.0.0",
     "@babel/core": "^7.2.3",
+    "@babel/plugin-proposal-class-properties": "^7.0.0",
+    "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "@babel/preset-react": "^7.0.0",
     "@commitlint/cli": "^6.1.3",
     "@commitlint/config-angular": "^6.1.3",
     "@edx/reactifex": "^1.1.0",
     "autoprefixer": "^7.2.5",
-    "babel-cli": "^6.26.0",
-    "babel-eslint": "^7.2.3",
-    "babel-jest": "^21.2.0",
+    "babel-core": "^7.0.0-bridge.0",
```

Comment:
We already have `@babel/core` in dependencies then why do we need to add `babel-core`?

Reply:
Yup good catch just removed it. May be was added by the script or by me while debugging and forgot to remove it... Anyway after removing still got the bundle when `npm run build`